### PR TITLE
Add named scopes and collections examples

### PR DESCRIPTION
**Pull request by maria-robobug**
https://issues.couchbase.com/browse/DOC-8160

Adds examples for named collections in KV and Query services

### CODE REVIEW

File: modules/howtos/examples/kv-counter.php [hunk 1]
```diff
@@ -1,32 +1,33 @@
...
+
 use \Couchbase\ClusterOptions;
 use \Couchbase\Cluster;
 use \Couchbase\IncrementOptions;
 use \Couchbase\DecrementOptions;
 
 $opts = new ClusterOptions();
 $opts->credentials("Administrator", "password");
-$cluster = new Cluster("couchbase://192.168.1.101", $opts);
+$cluster = new Cluster("couchbase://localhost", $opts);
```
Comment (RichardSmedley) [inline modules/howtos/examples/kv-counter.php:10]: LGTM - but did we move to 127.0.0.1 from localhost, for a problem on some VMs?

Comment (maria-robobug) [reply modules/howtos/examples/kv-counter.php:10]: Not 100% sure, I've been using localhost for most of these examples, I would assume if you were using VMs they would actually have an IP address though?

I can always change it to `127.0.0.1` to be sure, but would be good to know what is actually the issue as it worked fine for me with `localhost` (docker).

Comment (maria-robobug) [reply modules/howtos/examples/kv-counter.php:10]: @avsej do you have any suggestion about the above comment? We are trying to ensure that these examples can at least run against localhost. Is it preferable to use `127.0.0.1` or `localhost` in the connection string?

Comment (RichardSmedley) [reply modules/howtos/examples/kv-counter.php:10]: For reference, it's a TLS problem:
https://issues.couchbase.com/browse/DOC-6808
https://github.com/couchbase/docs-sdk-go/blob/release/2.1/modules/howtos/pages/managing-connections.adoc#secure-connections

Comment (maria-robobug) [reply modules/howtos/examples/kv-counter.php:10]: As per discussion in the last Friday SDK meeting(26th March), we can keep `localhost` in this case I believe.

**Pull request merged**